Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN CGO_ENABLED=0 GOOS=linux go build -o bin/machine-api-operator -a -ldflags '-
 
 # Final container
 FROM openshift/origin-base
-RUN yum install -y ca-certificates
+#RUN yum install -y ca-certificates
```

Comment:
do we need this? otherwise please remove commented code

Reply:
sure